counsel.el

```diff
@@ -6182,6 +6182,14 @@ The buffers are those opened during a session of `counsel-switch-buffer'."
   "When non-nil, `counsel-switch-buffer' will preview virtual buffers."
   :type 'boolean)
 
+(defcustom counsel-switch-buffer-display-empty-buffer-when-no-preview nil
+  "How `counsel-switch-buffer' behaves when not displaying a preview.
+
+When non-nil, `counsel-switch-buffer' will display an empty buffer when
+the current candidate is not previewed.  When nil, `counsel-switch-buffer'
+will instead display the buffer it was called from."
```

Comment:
Should this explain what it means for a current candidate's preview to not be displayed? E.g. when would that occur?